### PR TITLE
Add and remove functionalities added.

### PR DESCRIPTION
**Pull request by HatimDev-HE**
In this PR, I added the add and remove functions to my app.

### CODE REVIEW

File: src/modules/dev.js [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+        const inputField = event.target.parentNode.querySelector('p');
+        const dots = event.target.parentNode.parentNode.querySelector('.bi-three-dots-vertical');
+        const trash = event.target.parentNode.parentNode.querySelector('.bi-trash');
+        inputField.classList.toggle('line-through');
+        trash.classList.toggle('d-none');
+        dots.classList.toggle('d-none');
+        this.markCompleted(parseInt(event.target.id, 10));
+      });
+    });
+  }
```
Comment (elisha2kyakpo1) [inline src/modules/dev.js:63]: - [ ] Let's implement the editing functionally, it's part of this milestone. I couldn't edit a task description though everything works as expected.
`A function for editing task descriptions is implemented.`

File: src/modules/dev.js [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+// e.preventDefault();
+
+//   let newInput = new List(listInput.value, listItems.length + 1);
+
+//   const div = document.createElement('div');
+//   div.setAttribute('draggable', 'true');
+//   div.setAttribute('class', 'list');
+
+//   const check = document.createElement('input');
+//   check.setAttribute('type', 'checkbox');
```
Comment (elisha2kyakpo1) [inline src/modules/dev.js]: - [ ] Let's remove the commented code to make our codebase look clean.

**Review by elisha2kyakpo1 — CHANGES_REQUESTED** [review]
Hi 👋 @HatimDev-HE, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- The PR title and description looks professional👍
- No linter errors👍
- You have descriptive commit messages👍

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js [hunk 3]
```diff
@@ -1,57 +1,102 @@
...
+//     return listItems;
+//   }
+//   static addTask(task) {
+//     const tasks = this.getListItems();
+//     tasks.push(task);
+//     localStorage.setItem('listItems', JSON.stringify(tasks));
+//     const inputValue = document.querySelector('#new-item');
+//     inputValue.value = '';
+//   }
+//   static showItems(task) {
```
[the remark this answers is not in the document]
Comment (HatimDev-HE) [reply src/index.js]: done

File: src/modules/dev.js (hunk 1 above)
Comment (HatimDev-HE) [reply src/modules/dev.js:63]: done

File: src/modules/dev.js (hunk 2 above)
Comment (HatimDev-HE) [reply src/modules/dev.js]: done

File: README.md [hunk 4]
```diff
@@ -1,2 +1,60 @@
...
+
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
+
```
Comment (elmar8287) [inline README.md:34]: - [ ] [OPTIONAL] Please, consider removing the unused section or fulfill them accordingly 👍

File: README.md [hunk 5]
```diff
@@ -1,2 +1,60 @@
...
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```
Comment (elmar8287) [inline README.md:60]: - [ ] [OPTIONAL] Please, consider fixing the broken link to the MIT.md file 👍

**Review by elmar8287 — APPROVED** [review]
Hi, 

# APPROVED  👍 🥇 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

Congratulations! 🎉 

### Highlights

- Good implementation of suggestions from the reviewer 👍 

### Optional suggestions

- Please, consider my optional comments under the review, before merging 👍 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**